tests/portal/test_polls.py

```diff
@@ -188,12 +226,26 @@ def test_update_vote(self):
     def test_history(self):
         payload_1 = {"poll_options": [self.p1_op1_id]}
         self.client.post("/portal/votes/", payload_1)
-        response = self.client.get(reverse("portal:poll-history"))
+        response = self.client.get(reverse("portal:poll-history-list"))
         res_json = json.loads(response.content)
         # asserts that history works, can see expired posts and posts that
         # user voted for
         # also asserts that data collection works
         self.assertEqual(3, len(res_json[0]["poll"]["options"]))
-        self.assertEqual(1, res_json[0]["poll_statistics"][0]["breakdown"]["SEAS"])
         self.assertEqual(3, len(res_json[1]["poll"]["options"]))
-        self.assertEqual({}, res_json[1]["poll_statistics"][0]["breakdown"])
+
+    def test_recent_poll(self):
+        # answer poll
+        payload_1 = {"poll_options": [self.p1_op1_id]}
+        self.client.post("/portal/votes/", payload_1)
+        response = self.client.get(reverse("portal:poll-history-recent"))
+        res_json = json.loads(response.content)
+        # assert that payload1 poll answered is most recent
+        self.assertEquals(self.p1_op1_id, res_json["id"])
```

Comment:
i am not entirely sure this test case works (check out the github actions). i think it would be better to get the `id` from filtering for the most recent `PollVote` object, and then comparing it to `res_json['id']`